gcp_variant_transforms/testing/integration/run_tests.py

```diff
@@ -287,7 +288,10 @@ def _get_args():
       required=False)
   parser.add_argument('--include_large_tests',
                       type=bool, default=False, nargs='?', const=True,
-                      help='If set, runs the large_tests too.')
+                      help='If set, runs the large_tests and medium_tests too.')
```

Comment:
Suggestions: Since you have moved tests run in presubmit mode into a separate directory, maybe it makes sense to change this to `--run_all_presubmit_tests` instead of `--include_large_tests`. If you decided to do so then you should also update [cloudbuild_CI.yaml](https://github.com/googlegenomics/gcp-variant-transforms/blob/9ddddfe2f0f5b59268a8bac92864806eadca50e1/cloudbuild_CI.yaml#L49).

Reply:
Right, I am also thinking about changing this. Done now.